sql/core/src/main/scala/org/apache/spark/sql/execution/CacheManager.scala

```diff
@@ -328,4 +326,15 @@ class CacheManager extends Logging with AdaptiveSparkPlanHelper {
     if (needToRefresh) fileIndex.refresh()
     needToRefresh
   }
+
+  /**
+   * If CAN_CHANGE_CACHED_PLAN_OUTPUT_PARTITIONING is disabled, just return original session.
```

Comment:
"If CAN_CHANGE_CACHED_PLAN_OUTPUT_PARTITIONING is disabled" -> do we mean enabled?

Reply:
good catch!

Reply:
Sorry, forgot to fix the comment, create followup [#32543](https://github.com/apache/spark/pull/32543).